flake.nix

```diff
@@ -0,0 +1,74 @@
+{
+  description = "Command line tool for building web assets, wasm and mobile libraries";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+  };
+
+  outputs = { self, nixpkgs }: 
+    let
+      name = "builder";
+      version = "0.0.3";
+
+      packages = {
+        aarch64-darwin = {
+          label = "aarch64-darwin";
+          triple = "aarch64-apple-darwin";
+          checksum = "sha256-o82EeaeyppnCawV5F4pJNAsUlr2TEHHnHmQDyH9Ii9k=";
+          platform = "darwin";
+        };
+        x86_64-darwin = {
+          label = "x86_64-darwin";
+          triple = "x86_64-apple-darwin";
+          checksum = "todo";
```

Comment:
The checksum can be found in the github release. There it is in hex format, if hex is not supported by fetchurl, then you can convert it here: https://base64.guru/converter/encode/hex
Also: add a comment with instructions for how to find and set the checksums.